### DISCUSSION TITLE
Getting the Collection Values

### DISCUSSION THREAD

**Discussion by Corno1** [joalla/discogs_client#54] [Ideas] [answered]
Hey, 

I'm trying to fetch my collections values on an hourly base but at this time I think its not possible with this client version (or did i oversee sth?). Discogs has an Endpoint for that i created a workaround but i wanted to ask if its planned  in further versions to get the values with this client ?

Thanks in advance

**Comment by alifhughes** [comment]
Hey @Corno1 thanks for reaching out. I'm guessing you mean [this endpoint?](https://www.discogs.com/developers#page:user-collection,header:user-collection-collection-value) If so, you're right we haven't included it in our client yet, but I see no reason why we cannot. I'll try get this done for you this week if that is okay, feel free to submit a PR if you would like to give it a go yourself though, we always welcome contributions 🙂

**Reply by Corno1** [reply]
Hey @alifhughes, yeah exactly thats the endpoint I'm talking about. I didn't expect a solution this fast, so thank you very much helping so fast :). I tried to extend your files but this is my first Python project so think I couldn't deliver a perfect solution fitting your project as I'm usually an Kotlin developer for mobile solutions.

**Reply by alifhughes** [reply]
Pull request for this functionality is [here](https://github.com/joalla/discogs_client/pull/55), will get the guys to review it and hopefully we can release it asap 🙂

**Comment by alifhughes** [comment, marked as answer]
Hey @Corno1, hope you're well. [That PR](https://github.com/joalla/discogs_client/pull/55) has been merged and released now, so by downloading the latest version of the client you're able to get your own collection value 🙂 
- [Link to release](https://pypi.org/project/python3-discogs-client/2.3.12/)
- [Link to package](https://pypi.org/project/python3-discogs-client/)

This can be used like so:
```python
>>> import discogs_client
>>> d = discogs_client.Client("TestCollectionValue", user_token="...")
>>> me = d.identity()
>>> me.collection_value
<CollectionValue £1,000,000.10>
>>> print(me.collection_value.median, me.collection_value.maximum, me.collection_value.minimum)
£1,000,000.10 £2,000,000.10 £500,000.10
>>>
```

Remember you have to be authenticated to be able to access your collection value. Going to mark this as answered, please reopen if there is anything I've missed. Enjoy!

**Reply by Corno1** [reply]
First of all, I really thank you for your work and helping me out on this one :).

Thank you as well for the quick tutorial, I really appreciate your work!

**Reply by Corno1** [reply]
P.S. I tried it and everything works fine!

**Reply by alifhughes** [reply]
You're welcome!
